src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+	
+	public String getPrintableString();
+
+	/**
+	  * Returns a concatenated version of the printable strings of each object.
+	  */
+	public static String getPrintableString(Printable... printables){
+		StringBuilder builder = new StringBuilder();
+		builder.append(printables[0].getPrintableString());
+		for(int i=1;i<printables.length;i++){
```

Comment:
You could have used a `foreach` loop instead. Coding standard violation! missing/extra spaces.